### ISSUE TITLE
Latest version (0.11) broken

### ISSUE THREAD

**Issue by yhnavein** [petyosi/react-virtuoso#31]
When upgraded to the latest version (0.11), my app is not buildable by TypeScript because of the following error:

```
Property 'computeItemKey' is missing in type '{ totalCount: number; overscan: number; itemHeight: number; className: string; item: (index: number) => Element; }' but required in type 'Readonly<VirtuosoProps>'.
```

There is not a single mention about `computeItemKey` in the docs or the code examples

**Comment by petyosi** [comment]
Ugh, sorry for that, this should be optional (and documented). Will release a fix later today.

**Comment by petyosi** [comment]
Published 0.11.1 that fixes the typings
